blog/what-is-a-Slice.mdx

```diff
@@ -0,0 +1,47 @@
+
+If you see here, the term **application slice** or **Slice** is mentioned. This is one of the key components in KubeSlice and in this particular article we will understand what exactly is a Slice and how it enables KubeSlice to tackle the problems that Kubernetes have in achieving multi-tenancy on a multi-cluster architecture.
+
+So let’s start!
+
+### What is a Slice?
+
+One of the best real-world examples out there of a Slice is the concept of **Network Slicing** in 5G networks today. Network Slicing is a method of creating multiple unique **logical** and **virtualized** networks over a common (shared) infrastructure. You all might be familiar with remote Virtual Machines, right? Well, these are virtual servers running on the cloud or containers and are not physically present. Similarly, a network slice is carved out of a large and common infrastructure. 
+
+Each network slice is logically separated from the other, is an independent part of the network, and targets different services with different requirements at the same time. This enables to build and manage a network, which has the capabilities to meet and exceed the emerging requirements from a wide range of users and that too, at the same time: the concept of multi-tenancy.
+
+### Concept of a Slice in KubeSlice
+
+In terms of Kubernetes, a Slice is a Kubernetes [custom resource](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) which can be defined, and it basically describes a way to implement multi-tenancy in a cluster. Multi-tenancy is a concept wherein a single instance of an application or a single Kubernetes cluster can serve multiple users (called tenants) at the same time and is able to withstand that much amount of load.
+
+You might be wondering, how does the creation of a Slice an essential part of KubeSlice and helps in achieving the goals of multi-tenancy?
+
+Well, KubeSlice actually creates these logical application **slice boundaries** in a cluster, that allows pods and services to communicate efficiently within a cluster, across clusters as well as on the cloud or any other data center as well. In order to communicate in a multi-cluster architecture, an overlay network (an application slice) is created which provides seamless connectivity between the pods of an application running in multiple clusters.
+
+Therefore, due to this **slice overlay network**, pods are able to communicate seamlessly across cluster boundaries.
+
+You may ask a question here, what are some of the benefits that we get due to the creation of the slice overlay network?
+
+There are a lot of advantages achieved here:
+
+- **Better Resiliency** - In layman's language, Resiliency is the ability of an application to react to problems in one of its components and still provide the best possible service. By deploying an application across clusters and connecting using the slice overlay network, application services get better resiliency
```

Comment:
Could we go a bit further and explain how they become more resilient? In the sense, how does the overlay network actually help with bettering resiliency? Does it provide isolation/failover capabilities? We need to explain that here.
```suggestion
- **Better Resiliency** - Resiliency is the ability of an application to react to problems in one of its components and still provide the best possible service. By deploying an application across clusters and connecting using the slice overlay network, application services are able to achieve resiliency.
```